pkgs/development/python-modules/transformers/default.nix

```
@@ -1,41 +1,76 @@
 { buildPythonPackage
 , stdenv
 , fetchFromGitHub
-, sacremoses
-, requests
-, sentencepiece
 , boto3
-, tqdm
+, filelock
 , regex
+, requests
 , numpy
+, sacremoses
+, sentencepiece
+, timeout-decorator
+, tokenizers
+, tqdm
 , pytest
```

```suggestion
, pytestCheckHook
```
I suggest using pytestCheckHook. Makes these disables cleaner.

---

Great idea! Looks much better now, thanks!